source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRRegistration.h

```diff
@@ -18,6 +18,7 @@
 #endif
 #import "ACOBaseCardElement.h"
 #import "ACRIFeatureFlagResolver.h"
+#import "ACRIImageResolver.h"
```

Comment:
Use forward references. 
While you are adding that also examine the existing imports and move them to forward references.